docs/host.md

```diff
@@ -1,118 +0,0 @@
-# Landing OpenVINO&trade; Model Server on Bare Metal Hosts and Virtual Machines
```

Comment:
why is it dropped? We need information how to use OVMS without docker containers. It was one of the announced features. This page is also referenced in external documentation like https://docs.openvinotoolkit.org/latest/openvino_docs_ovms.html

Reply:
It's dropped because of multiple issues with this document: the instructions are missing some of the dependencies, libraries and drivers. Additionally it was agreed with PM to support only docker deployment. The bare metal version is not validated in any way.